src/SonarLint.VSSpecificAssemblies.props

```diff
@@ -182,6 +182,7 @@
     <ItemGroup>
       <ReferencePath Remove="@(ReferencePath)" Condition="'%(FileName)' == 'Microsoft.VisualStudio.Shell.14.0'" />
       <ReferencePath Remove="@(ReferencePath)" Condition="'%(FileName)' == 'Microsoft.VisualStudio.Shell.Immutable.14.0'" />
+      <ReferencePath Remove="@(ReferencePath)" Condition="'%(FileName)' == 'Microsoft.VisualStudio.Shell.Immutable.10.0'" />
```

Comment:
SVsServiceProvider is used in IssueViz and in shell 14's version it's coming from `Microsoft.VisualStudio.Shell.Immutable.10.0` (it seems like [it's not part of Immutable.14](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.svsserviceprovider?view=visualstudiosdk-2015)).

So we need to remove it, since we already have one in shell 15's version in `Microsoft.VisualStudio.Shell.Framework`.